client/extensions/woocommerce/app/products/product-form-simple-card.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormCurrencyInput from 'components/forms/form-currency-input';
+import FormDimensionsInput from '../../components/form-dimensions-input';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+
+const ProductFormSimpleCard = ( { product, editProduct, translate } ) => {
+	const setDimension = ( e ) => {
+		const dimensions = { ...product.dimensions, [ e.target.name ]: e.target.value };
+		editProduct( product, { dimensions } );
+	};
+
+	const setWeight = ( e ) => {
+		editProduct( product, { weight: e.target.value } );
+	};
+
+	const setPrice = ( e ) => {
+		editProduct( product, { regular_price: e.target.value } );
+	};
+
+	const toggleStock = () => {
+		editProduct( product, { manage_stock: ! product.manage_stock } );
+	};
+
+	const setStockQuantity = ( e ) => {
+		editProduct( product, { stock_quantity: e.target.value } );
+	};
+
+	const setBackorders = ( e ) => {
+		editProduct( product, { backorders: e.target.value } );
+	};
+
+	// TODO Pull in currency and currency position.
+	const renderPrice = () => (
+		<Card className="products__product-form-price">
+			<FormLabel>{ translate( 'Price' ) }</FormLabel>
+			<FormCurrencyInput
+				currencySymbolPrefix="$"
+				name="price"
+				value={ product.regular_price || '' }
+				onChange={ setPrice }
+				size="4"
+			/>
+		</Card>
+	);
+
+	// TODO pull in dimension and weight units.
+	const renderDeliveryDetails = () => (
+		<Card className="products__product-form-delivery-details">
+			<div className="products__product-dimensions-weight">
+				<FormFieldSet className="products__product-dimensions-input">
+					<FormLabel>{ translate( 'Dimensions' ) }</FormLabel>
+					<FormDimensionsInput
+						unit={ translate( 'in' ) }
+						dimensions={ product.dimensions }
+						onChange={ setDimension }
+					/>
+				</FormFieldSet>
+				<FormFieldSet className="products__product-weight-input">
+					<FormLabel>{ translate( 'Weight' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Weight', { context: 'Dns Record'} )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woocommerce-simple-product-fields)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).